src/README.md

```diff
@@ -127,13 +127,20 @@ file inside your source code repository.
     Pipenv.){% endif %}
 
 
+* **DISABLE_SETUP_PY_PROCESSING**
```

Comment:
AFAIK the list of envvars used to be alphabetized and still is besides this and ENABLE_PIPENV. Could you please put these two in the alphabetical order?